LitElement conversion part 3: theme and visual tests

web-padawan opened the pull request:
Connected to #128

DiegoCardoso commented:
There's a difference of how the navbar shadow is working related to the drawer in Material between default and touch-optimized:

#### default
![Screenshot 2020-02-12 at 16 48 35](https://user-images.githubusercontent.com/262432/74345957-855c6400-4db7-11ea-8e14-b808b75f0059.png)

#### touch-optmized
![Screenshot 2020-02-12 at 16 49 09](https://user-images.githubusercontent.com/262432/74346003-9907ca80-4db7-11ea-9e5b-650438c99620.png)

web-padawan commented:
@DiegoCardoso thanks, good catch. Fixed.

We already had a fix for that (using `z-index`)  but it only worked for top navbar. So I changed the DOM structure to put drawer after both navbars. This way we ensure its `z-index` wins.

The drawer must be positioned with `position: absolute` anyways, so its place in the DOM does not affect anything than `z-index`. And I wanted to avoid having `z-index: 2` because of that.
